src/tokens/eth/0x7865af71cf0b288b4E7F654f4F7851EB46a2B7F8.json

```diff
@@ -0,0 +1,37 @@
+{
+"symbol": "SNTVT",
+
+"name": "Sentivate",
+
+"type": "ERC20",
+
+"address": "0x7865af71cf0b288b4E7F654f4F7851EB46a2B7F8",
+"ens_address": "",
+"decimals": "18",
```

Comment:
Please change this to an integer instead.

Reply:
changed to integer

Reply:
the decimal property is still a string.